chain-abci/src/enclave_bridge/edp/tdbe.rs

```diff
@@ -42,38 +43,43 @@ pub struct TdbeApp {
     remote_rpc_address: Option<String>,
     /// Local TDBE server address to listen on. E.g. `127.0.0.1:3445`
     local_listen_address: String,
+    /// UDS to push secrets to tx-validation enclave
+    tve_stream: UnixStream,
 }
 
 impl TdbeApp {
     /// Creates a new instance of TDBE app
     pub fn new(
         tdbe_config: &TdbeConfig,
         ra_config: &SpRaConfig,
-        storage: Arc<dyn KeyValueDB>,
+        _storage: Arc<dyn KeyValueDB>,
+        tve_stream: UnixStream,
     ) -> std::io::Result<Self> {
         // - `chain_abci_stream` is passed to enclave. Encalve can send requests to chain-abci
         //   using this
         // - `chain_abci_receiver` listens to the requests sent by enclave and responds to them
-        let (chain_abci_stream, chain_abci_receiver) = UnixStream::pair()?;
+        let (chain_abci_stream, _chain_abci_receiver) = UnixStream::pair()?;
 
         // - `persistence_stream` is passed to enclave. Encalve can send requests to chain-storage
         //   using this
         // - `persistence_receiver` listens to the requests sent by enclave and responds to them
-        let (persistence_stream, persistence_receiver) = UnixStream::pair()?;
+        let (persistence_stream, _persistence_receiver) = UnixStream::pair()?;
 
-        spawn_chain_abci_thread(chain_abci_receiver, storage.clone());
-        spawn_persistence_thread(persistence_receiver, storage);
+        // FIXME: spawn these when they actually do something
+        // spawn_chain_abci_thread(chain_abci_receiver, storage.clone());
```

Comment:
Why is `chain_abci_thread` removed? Isn't it supposed to spawn a new thread?

Reply:
it's not removed, it's commented out, as it's unused in this step